### PR TITLE
enable all target mock documents

### LINKED ISSUE #133: Support multiple target Document in UI
UI supports multiple source Document, but not for target Document. Multiple target Document is also required to support producing properties/headers kind of thing apart from main payload Document.
datamapper

### PR DESCRIPTION
**Pull request by akieling**
fix #133

### CODE REVIEW

**Comment by igarashitm** [comment]
OK cool, so multiple target document is already supported.
I checked in my local, and I just remember the instance documents are almost duplicate. Also the `Name` document doesn't look useful. I'm sorry to change my mind, but can you disable
* XMLInstanceSource
* JSONInstanceSource
* Name
for both of source and target?

**Comment by igarashitm** [comment]
Also, can you adjust the commit log just like we do in syndesis, so we can track it from commit log later?
```
chore: Enable all target mock documents

Fixes: #133
```

**Comment by akieling** [comment]
Done

**Comment by igarashitm** [comment]
Thanks, but can you also disable `addMockJSONSources` and `addMockJSONTarget` ? it's just an alias of `addMockJSONInstanceSources`

File: ui/src/app/lib/atlasmap-data-mapper/components/data.mapper.example.host.component.ts [hunk 1]
```diff
@@ -97,17 +97,17 @@ export class DataMapperAppExampleHostComponent implements OnInit {
...
         c.initCfg.addMockJSONSchemaSources = true;
 
-        c.initCfg.addMockJavaTarget = false;
+        c.initCfg.addMockJavaTarget = true;
         c.initCfg.addMockJavaCachedTarget = false;
-        c.initCfg.addMockXMLInstanceTarget = false;
-        c.initCfg.addMockXMLSchemaTarget = false;
-        c.initCfg.addMockJSONTarget = false;
-        c.initCfg.addMockJSONInstanceTarget = false;
+        c.initCfg.addMockXMLInstanceTarget = true;
```
Comment (igarashitm) [inline ui/src/app/lib/atlasmap-data-mapper/components/data.mapper.example.host.component.ts]: can you disable instance document?

File: ui/src/app/lib/atlasmap-data-mapper/components/data.mapper.example.host.component.ts [hunk 2]
```diff
@@ -97,17 +97,17 @@ export class DataMapperAppExampleHostComponent implements OnInit {
         c.initCfg.addMockJavaSources = true;
         c.initCfg.addMockJavaCachedSource = false;
         c.initCfg.addMockXMLInstanceSources = false;
-        c.initCfg.addMockXMLSchemaSources = false;
-        c.initCfg.addMockJSONSources = false;
+        c.initCfg.addMockXMLSchemaSources = true;
+        c.initCfg.addMockJSONSources = true;
```
Comment (igarashitm) [inline ui/src/app/lib/atlasmap-data-mapper/components/data.mapper.example.host.component.ts]: This is just an alias of JSON Instance Sources, so can you also disable?

File: ui/src/app/lib/atlasmap-data-mapper/components/data.mapper.example.host.component.ts [hunk 3]
```diff
@@ -97,17 +97,17 @@ export class DataMapperAppExampleHostComponent implements OnInit {
...
+        c.initCfg.addMockJavaTarget = true;
         c.initCfg.addMockJavaCachedTarget = false;
-        c.initCfg.addMockXMLInstanceTarget = false;
-        c.initCfg.addMockXMLSchemaTarget = false;
-        c.initCfg.addMockJSONTarget = false;
-        c.initCfg.addMockJSONInstanceTarget = false;
+        c.initCfg.addMockXMLInstanceTarget = true;
+        c.initCfg.addMockXMLSchemaTarget = true;
+        c.initCfg.addMockJSONTarget = true;
+        c.initCfg.addMockJSONInstanceTarget = true;
```
Comment (igarashitm) [inline ui/src/app/lib/atlasmap-data-mapper/components/data.mapper.example.host.component.ts]: can you disable instance document?

File: ui/src/app/lib/atlasmap-data-mapper/components/data.mapper.example.host.component.ts [hunk 4]
```diff
@@ -97,17 +97,17 @@ export class DataMapperAppExampleHostComponent implements OnInit {
...
-        c.initCfg.addMockJavaTarget = false;
+        c.initCfg.addMockJavaTarget = true;
         c.initCfg.addMockJavaCachedTarget = false;
-        c.initCfg.addMockXMLInstanceTarget = false;
-        c.initCfg.addMockXMLSchemaTarget = false;
-        c.initCfg.addMockJSONTarget = false;
-        c.initCfg.addMockJSONInstanceTarget = false;
+        c.initCfg.addMockXMLInstanceTarget = true;
+        c.initCfg.addMockXMLSchemaTarget = true;
+        c.initCfg.addMockJSONTarget = true;
```
Comment (igarashitm) [inline ui/src/app/lib/atlasmap-data-mapper/components/data.mapper.example.host.component.ts]: This is just an alias of JSON Instance target, so can you also disable?

**Comment by akieling** [comment]
Done

**Pull request merged**